src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -2542,5 +2547,12 @@ public static FormulaValue IsBlankOrError(IRContext irContext, FormulaValue[] ar
 
             return new BooleanValue(irContext, false);
         }
+
+        public static async ValueTask<FormulaValue> PowerFxVersion(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            IEnumerable<Assembly> powerFxAssemblies = AppDomain.CurrentDomain.GetAssemblies().Where(a => a.FullName.StartsWith("Microsoft.PowerFx", StringComparison.OrdinalIgnoreCase));

```

Comment:
Do not call AppDomain.CurrentDomain  or anything on AppDomain.  Consider it a banned API.

Reply:
What alternate API do you suggest?